daterangepicker.js

```diff
@@ -1142,7 +1151,10 @@
 
             $(document).off('.daterangepicker');
             $(window).off('.daterangepicker');
-            this.container.hide();
+            this.container.delay(200).hide({
```

Comment:
What is the reason of the delay? If for something else than pure UI, maybe leave a comment for context?